rasa/shared/core/events.py

```diff
@@ -686,21 +688,22 @@ def __eq__(self, other) -> bool:
         return self.use_text_for_featurization == other.use_text_for_featurization
 
 
-class DefinePrevUserUtteredEntities(DefinePrevUserUttered):
-    """Event that is used to set entities on a previous user uttered event."""
+class EntitiesAdded(DefinePrevUserUttered):
```

Comment:
I'd say the super class `DefinePrevUserUttered` should be removed, what do you think?

Reply:
kept it but made it a more general approach to deduplicate this behavior for other events as well